### PR TITLE
`Signup` CTAs around /fr pages

### PR DESCRIPTION
**Pull request by samjewell**
## Home
screen shot 2015-06-22 at 17 37 58

screen shot 2015-06-22 at 17 38 09

## /features
screen shot 2015-06-22 at 17 38 21

screen shot 2015-06-22 at 17 38 32

## /tarifs
screen shot 2015-06-22 at 17 38 41

screen shot 2015-06-22 at 17 38 48

## /securite
screen shot 2015-06-22 at 17 38 57

screen shot 2015-06-22 at 17 39 03

### CODE REVIEW

**Comment by samjewell** [comment]
@octaveauger would you take a look at the Francais? Cheers m8

**Comment by samjewell** [comment]
@jackfranklin would you be up for reviewing this? Thanks man :-)

File: app/components/header/header.js [hunk 1]
```diff
@@ -152,20 +153,32 @@ class Header extends React.Component {
...
                 <li className='u-pull-start'>
-                  <a href='/users/sign_in' id='track-nav-sign-in'
-                  className={classNames('nav-btn btn btn--small u-text-light u-text-xxs u-relative',
-                    'u-text-transform-none u-text-no-smoothing', {
-                    'btn--invert-hollow': isInverted,
-                    'btn--hollow': !isInverted,
-                  })}>
-                    <Message pointer='header.login_btn' />
-                  </a>
+                  <Translation locales='en'>
```
Comment (Ken-Lo) [inline app/components/header/header.js]: Here, you could put the signup link URL as a message on the locale and use `<Href to='message_name'>`. We use this whenever we link to the API docs e.g.
e.g. https://github.com/gocardless/splash-pages/blob/dev/app/pages/faq/merchants/developer-api/faq-merchants-developer-api.en.js#L19

File: app/components/start-taking-payments/start-taking-payments.js [hunk 2]
```diff
@@ -10,10 +12,21 @@ export default class StartTakingPaymentsCTA extends React.Component {
         <div className='site-container u-text-center u-padding-Vxxl'>
           <div className='u-padding-Vxl'>
             <h2 className='u-text-heading u-color-heading u-text-light u-text-l u-margin-Bm'>
-              Sign up in minutes, take payments today
+              <Message pointer='signup_cta.main' />
             </h2>
-            <a href='/merchants/new' className='btn'>Start taking payments</a>
-            <p className='u-color-p u-margin-Ts'>No set up costs, no hidden charges, no commitments</p>
+            <Translation locales='en'>
```
Comment (Ken-Lo) [inline app/components/start-taking-payments/start-taking-payments.js]: As above with `<Href to>` should greatly simplify this code block :)

File: app/components/header/header.js [hunk 3]
```diff
@@ -86,15 +87,15 @@ class Header extends React.Component {
                   </Link>
                 </IfLinkExists>
               </div>
-              <div className='nav__item u-relative'>
-                <IfLinkExists to='stories'>
+              <IfLinkExists to='stories'>
+                <div className='nav__item u-relative'>
```
Comment (samjewell) [inline app/components/header/header.js]: move classes up onto `IfLinkExists`

File: app/components/header/header.js [hunk 4]
```diff
@@ -152,20 +153,32 @@ class Header extends React.Component {
                 </Popover>
               </div>
             </nav>
-            <ul className='u-cf u-pull-start'>
-              <Translation locales='en-GB'>
+            <IfLocale hasInstantSignup>
+              <ul className='u-cf u-pull-start'>
```
Comment (samjewell) [inline app/components/header/header.js]: move the classes up again here

File: app/components/header/header.js [hunk 5]
```diff
@@ -86,15 +87,15 @@ class Header extends React.Component {
                   </Link>
                 </IfLinkExists>
```
Comment (samjewell) [inline app/components/header/header.js]: do same here too

**Comment by Ken-Lo** [comment]
Nice one @samjewell!

Just wondering if you knew why it does this in Chrome?
weird buttons

**Comment by octaveauger** [comment]
@samjewell looks all good to me - I also updated a couple of other things where we explained how to get started.
As @Ken-Lo mentions the header is broken on chrome.

**Comment by jackfranklin** [comment]
@samjewell just pushed a commit that should fix the heading :+1:

**Comment by octaveauger** [comment]
@jackfranklin not fixed entirely yet
image

**Comment by samjewell** [comment]
@Ken-Lo @jackfranklin @octaveauger Thanks for all of the feedback, should all be fixed up now. If it passes tests I'd like to merge it into Dev, and check everything on staging, if that's ok?

**Comment by octaveauger** [comment]
Looks good to me now!

Octave Auger
co-Directeur France
+33 (0) 9 87 67 94 20 | +44 (0) 7450 245 225
www.gocardless.fr
Ils parlent de GoCardless: Le Figaro
<http://www.lefigaro.fr/societes/2015/04/12/20005-20150412ARTFIG00171-gocardless-pret-a-bousculer-les-banques-francaises.php>
| Les Echos
<http://www.lesechos.fr/finance-marches/banque-assurances/0213743712-le-prelevement-bancaire-seduit-de-nouveaux-acteurs-1110875.php>
| Challenges
<http://www.challenges.fr/finance-et-marche/20150413.CHA4873/ces-start-up-qui-font-valser-les-modeles-de-la-city-de-londres.html>
| La Tribune
<http://www.latribune.fr/entreprises-finance/banques-finance/gocardless-ou-quand-la-fintech-londonienne-debarque-en-france-469023.html>
| Journal du net
<http://www.journaldunet.com/web-tech/start-up/fintech-gocardless-lancement-france.shtml>
|
BFM TV
<http://bfmbusiness.bfmtv.com/mediaplayer/video/start-up-co-gocardless-la-fintech-qui-veut-revolutionner-les-prelevements-bancaires-pour-les-entreprises-2204-508599.html>

**Comment by jackfranklin** [comment]
@samjewell merging and checking on staging sounds sensible :+1:

**Pull request merged**